source/tyr/tests/integration/users_test.py

```diff
@@ -494,3 +504,49 @@ def test_deletion_keys_and_auth(create_instance, mock_rabbit):
                            content_type='application/json')
     assert len(resp_auth['authorizations']) == 0
     assert mock_rabbit.called
+
+
+def test_get_user_with_shape(create_user, geojson_feature_collection):
+    """
+    We start by creating the user with a shape,
+    and we test that the attribute shape is returned and has_shape = True
+    """
+    resp = api_get('/v0/users/{}'.format(create_user))
+
+    assert resp['has_shape'] == True
+    assert resp['shape'] == geojson_feature_collection
+
+
+def test_get_user_with_shape_and_disable_geojson_param_true(create_user, geojson_feature_collection):
```

Comment:
would be nice to test the same but not providing param, see that we default to `disable_geojson=True`